pdc/apps/unreleasedvariant/apps.py

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2018 Red Hat
+# Licensed under The MIT License (MIT)
+# http://opensource.org/licenses/MIT
+#
+from django.apps import AppConfig
+
+
+class UnreleasedVariantConfig(AppConfig):
+    name = 'pdc.apps.unreleasedvariant'
+
+    def ready(self):
+        from pdc.apps.utils.utils import connect_app_models_pre_save_signal
+        connect_app_models_pre_save_signal(self)
```

Comment:
Again, since there are no models, there should also be no signals.

Reply:
I'll remove it